src/test/java/org/broadinstitute/consent/http/service/DaaServiceTest.java

```diff
@@ -213,8 +213,8 @@ void testSendDaaRequestEmailsWithSigningOfficials() throws Exception {
     when(user.getInstitutionId()).thenReturn(1);
 
     SimplifiedUser signingOfficial = mock(SimplifiedUser.class);
```

Comment:
IMO this is not a good use of `mock()`, since `SimplifiedUser` is a POJO
```suggestion
    SimplifiedUser signingOfficial = new SimplifiedUser();
```